src/firebase.js

```diff
@@ -0,0 +1,20 @@
+// Import the functions you need from the SDKs you need
+import { initializeApp } from "firebase/app";
```

Comment:
I like that you saved the firebase config into its own file, it keeps things much cleaner.